package.json

```diff
@@ -29,6 +29,7 @@
     "fuse.js": "^6.6.2",
     "lodash": "^4.17.21",
     "middleware-async": "^1.3.6",
+    "node-fetch-native": "^1.4.1",
```

Comment:
I don't see this used? Why don't we simply use the `fetch` which comes with node?